ax/ax_wait.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import time
+
+import acos_client
+
+sys.path.append(os.environ['HOME'])
+
+import config
+
+d = config.devices['ax-lsi']
+
+TIME_TO_WAIT=6 #00
```

Comment:
E225 - Missing whitespace around operator.
E261 - at least 2 spaces before inline comment
E262 - Inline comment should start with "# " (the space was emphasized)